test/internal/cleanup_test.go

```diff
@@ -76,16 +76,19 @@ func TestCleanup(t *testing.T) {
 	if orgID, set := os.LookupEnv("MONGODB_ATLAS_ORG_ID"); set {
 		args = append(args, "--orgId", orgID)
 	}
+
 	cmd := exec.Command(cliPath, args...)
 	cmd.Env = os.Environ()
 	resp, err := RunAndGetStdOut(cmd)
 	req.NoError(err, string(resp))
 	var projects admin.PaginatedAtlasGroup
 	req.NoError(json.Unmarshal(resp, &projects), string(resp))
 	t.Logf("projects:\n%s\n", resp)
+
 	for _, project := range projects.GetResults() {
 		projectID := project.GetId()
 		if projectID == os.Getenv("MONGODB_ATLAS_PROJECT_ID") {
+			t.Log("skip deleting default project", projectID)
```

Comment:
from the logs, we're not detecting this env variable and not skipping the projcect succesfully